keyboards/knobgoblin/keymaps/via/keymap.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2020 MrT1ddl3s
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT_ortho(
+                  KC_EQL, KC_PSLS, KC_PAST, KC_PMNS,
+                  KC_P7,  KC_P8,   KC_P9,   KC_PPLS,
+                  KC_P4,  KC_P5,   KC_P6,   KC_PPLS,
+         KC_MPLY, KC_P1,  KC_P2,   KC_P3,   KC_PENT,
+         KC_MUTE, MO(1),  KC_P0,   KC_PDOT, KC_PENT
+				 ),
```

Comment:
I'd wrap these in `/* clang-format off */` and `/* clang-format on */` just to be safe, so they don't get mangled next time someone decides to run `clang-format` on the entire codebase. Seems to happen once every few months.

Reply:
e.g.
```
    [0] = LAYOUT_ortho(/* clang-format off */
                  KC_EQL, KC_PSLS, KC_PAST, KC_PMNS,
                  KC_P7,  KC_P8,   KC_P9,   KC_PPLS,
                  KC_P4,  KC_P5,   KC_P6,   KC_PPLS,
         KC_MPLY, KC_P1,  KC_P2,   KC_P3,   KC_PENT,
         KC_MUTE, MO(1),  KC_P0,   KC_PDOT, KC_PENT
				 ), /* clang-format on */
```